tests/core/test_asr_metrics.py

```diff
@@ -119,8 +117,11 @@ def get_wer(prediction: str, reference: str):
             n2 = random.randint(0, 512)
             s1 = __randomString(n1)
             s2 = __randomString(n2)
-            logging.debug(n1)
-            logging.debug(s1)
-            logging.debug(n2)
-            logging.debug(s2)
-            self.assertEqual(get_wer(prediction=s1, reference=s2), word_error_rate(hypotheses=[s1], references=[s2]))
+
+            assert get_wer(prediction=s1, reference=s2) == word_error_rate(hypotheses=[s1], references=[s2])
```

Comment:
Please do a range check here instead of a hardcoded == so we can account for this unstable behaviour. Even better if we can remove this instability entirely.

Reply:
@tkornuta-nvidia For now please the mark the test with pleasefixme mark so that @okuchaiev can take a look.

Reply:
Actually I haven't _touched_ the `@pleasefixme` decorator... Seems that @blisc was playing with it too, then I had to solve conflicts and it was gone. Afterwards I have reverted it.

Reply:
Need to think about this one carefully - there is no randomness here. The mismatch could be due to floating operations order only, so it either matches exactly or the range should be pretty tight. Please leave as == for now